contracts/contracts/vault/VaultStorage.sol

```diff
@@ -99,7 +99,7 @@ contract VaultStorage is Initializable, Governable {
     /// @notice OToken mints over this amount automatically allocate funds. 18 decimals.
     uint256 public autoAllocateThreshold;
     /// @notice OToken mints over this amount automatically rebase. 18 decimals.
-    uint256 public rebaseThreshold;
+    uint256 public rebaseThreshold = 0;
```

Comment:
This value is actually used from the storage slot on the proxy, while the set here would happen in the constructor on the implementation contract. Because this would be doing something unintuitive, we don't do constructor time variable sets on these storage slots.

Reply:
I put this in to shut slither up. See the error below.

I should have looked deep for the underlying issue which is `VaultCore` was not inheriting from `VaultInitializer` that has the `initialize` function for the Vault. I've changed both `VaultCore` and `VaultAdmin` to inherit from `VaultInitializer`.

This raises the question of whether the `initialize` function should be on `VaultStorage`. The `initialize` function is closely linked to the storage variables so I would vote to just move `initialize` to the `VaultStorage` and get rid of `VaultInitializer`.

So Slither was right to complain!

```
VaultStorage.rebaseThreshold (contracts/vault/VaultStorage.sol#102) is never initialized. It is used in:
        - VaultCore.mint(address,uint256,uint256) (contracts/vault/VaultCore.sol#61-97)
        - VaultCore._redeem(uint256,uint256) (contracts/vault/VaultCore.sol#153-216)
Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#uninitialized-state-variables
```

Reply:
I would rather we not inherit from VaultInitializer or place VaultInitializer into vault storage. Want to keep the contract space in both contracts, since this has been a long term constraint in the past. Can we just use a slither comment to quite this message?

Reply:
I can live with it going in just one of the two contracts.

Reply:
I've changed `VaultAdmin` to inherit directly from `VaultStorage`